src/mono/wasm/debugger/tests/ApplyUpdateReferencedAssemblyChineseCharInPathㄨ/MethodBody1_v2.cs

```diff
@@ -0,0 +1,106 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System.Diagnostics;
+using System;
+//keep the same line number for class in the original file and the updates ones
+namespace ApplyUpdateReferencedAssembly
+{
+    public class MethodBody1 {
+        public static string StaticMethod1 ()
+        {
+            Console.WriteLine("v2");
+            bool c = true;
+            Debugger.Break();
+            return "NEWEST STRING";
+        }
+    }
+
+    public class MethodBody2 {
+        public static string StaticMethod1 () {
+            Console.WriteLine("original");
+            int a = 10;
+            Debugger.Break();
+            return "OLD STRING";
+        }
+    }
+
+    public class MethodBody3 {
+        public static string StaticMethod3 () {
+            bool c = true;
+            int d = 10;
+            int e = 20;
+            int f = 50;
+            return "NEWEST STRING";
+        }
+    }
+
+    public class MethodBody4 {
+        public static void StaticMethod4 () {
+        }
+    }
+
+
+
+
+
+
```

Comment:
Are these deliberate?

Reply:
yes, I don't want to move the line of definition of MethodBody5.